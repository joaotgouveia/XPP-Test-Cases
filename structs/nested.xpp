let a = { b = { c = 1 } };
println(a);
println(a.b);
a.b.c;;

let a = { b = { c = 1; d = 2}; c = 3 };
println(a.b.c);
println(a.b.d);
a.c;;
