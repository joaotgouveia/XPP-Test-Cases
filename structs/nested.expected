Value: 1
Type: int
Printed: record { b = record { c = 1 } } | record { c = 1 }

Value: 3
Type: int
Printed: 1 | 2
